jwst/master_background/tests/test_master_background.py

```diff
@@ -45,7 +45,7 @@ def science_image():
     image.meta.wcsinfo.roll_ref = 0
     image.meta.wcsinfo.ra_ref = 0
     image.meta.wcsinfo.dec_ref = 0
-    image = AssignWcsStep.call(image)
+    image = AssignWcsStep.call(image, sip_approx=False)
```

Comment:
The above comment applies here as well. This is an LRS (i.e. spectral observation).